lib/rules/selector-max-attribute/README.md

```diff
@@ -0,0 +1,70 @@
+# selector-max-attribute
+
+Limit the number of attributes in a selector.
+
+```css
+    [rel="external"] {}
+/** ↑
+ * This type of selector */
+```
+
+This rule resolves nested selectors before counting the number of classes in a selector. Each selector in a [selector list](https://www.w3.org/TR/selectors4/#selector-list) is evaluated separately.
+
+The `:not()` pseudo-class is also evaluated separately. The rule processes the argument as if it were an independent selector, and the result does not count toward the total for the entire selector.
+
+## Options
+
+`int`: Maximum attribute allowed.
+
+For example, with `2`:
+
+The following patterns are considered warnings:
+
+```css
+[type="number"][name="quality"][data-attribute="value"] {}
+```
+
+```css
+[type="number"][name="quality"][disabled] {}
+```
+
+```css
+[type="number"][name="quality"] {
+  & [data-attribute="value"] {}
+}
+```
+
+```css
+[type="number"][name="quality"] {
+  & [disabled] {}
+}
+```
+
+```css
+[type="number"][name="quality"] {
+  & > [data-attribute="value"] {}
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+[type="text"] {}
+```
+
+```css
+[type="text"][name="message"] {}
+```
+
+```css
+[type="text"][disabled]
+```
+
+```css
+[type="text"][name="message"],
+[type="number"][name="quality"] {} /* each selector in a selector list is evaluated separately */
+```
+
+```css
```

Comment:
@jeddy3 not, example of list above, this example show attributes selectors and attributes selectors in `not` calculate separately